### ISSUE TITLE
Remove normative references to RFC 6241

### ISSUE THREAD

**Issue by kwatsen** [netmod-wg/yang-next#12]
The YANG specification should not have any normative references to RFC 6241.

Other than the references from the sections to be removed via issue #11, the only other substantive references are in the Terminology section, which references the following form RFC 6241:

   o  configuration data
   o  configuration datastore
   o  datastore
   o  state data

These should probably be moved to the datastore RFC...

**Comment by schoenw** [comment]
Yes. This can hopefully be resolved with normative references to NMDA. ;-)

**Comment by abierman** [comment]
do not support this work
do not agree with the premise that an RFC reference is a real problem to solve
